pkg/catalog/catalog_test.go

```diff
@@ -58,16 +58,15 @@ var _ = Describe("Catalog", func() {
 
 		When("version is set to latest", func() {
 			It("returns the latest version", func() {
-				profiles := []profilesv1.ProfileDescription{{Name: "foo", Version: "v0.1.0"}, {Name: "foo", Version: "0.2.0"}, {Name: "bar", Version: "0.3.0"}, {Name: "foo"}}
+				profiles := []profilesv1.ProfileDescription{{Name: "foo", Version: "v0.1.0"}, {Name: "foo", Version: "v0.2.0"}, {Name: "bar", Version: "v0.3.0"}, {Name: "foo"}}
```

Comment:
I think we purposely had a mix of `v` and without, just for sanity checking :smile:

Reply:
ah, I see. :D Okay :D